Contents/mods/Perdition/media/scripts/atms.txt

```diff
@@ -59,4 +59,14 @@ module ATM
         WorldStaticModel = MoneyTrader,
         DisplayCategory	= Money,
     }
+	item BookTrader
+        {
+        Weight          = 60.0,
+        Type            = Normal,
+        DisplayName     = Book Vendor,
+        icon            = BookTrader,
+		StaticModel = BookTrader,
```

Comment:
We do not have this model, you could use the model of something else in the meantime